src/test/resources/opa.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM alpine:latest
+
+RUN apk add go git nginx bash
+
+ADD nginx.conf /etc/nginx/nginx.conf
+
+ADD entrypoint.sh /entrypoint.sh
+
+# Install the latest version of OPA from source, since we need nginx in the
+# container, OPA isn't packaged by Alpine, and the upstream OPA package
+# contains only OPA and no userland.
+RUN mkdir -p /src && \
```

Comment:
Hmm. Could we not grab the binary from the OPA package?
```dockerfile
COPY --from=openpolicyagent/opa:latest-static /opa ./opa
```
We could then drop the go+git dependency, and perhaps even use a standard nginx image.

Reply:
Building it from source was my way of making it architecture independent, but it looks like the upstream container has builds for both amd64 and arm64, so that should also work. I wasn't aware you could use COPY in this way, but assuming it works I think this will be a good improvement, cloning the repo and building from source adds a lot of time to the tests.

Reply:
That worked nicely, adopted this approach in c3136e3.